src/main/msp/msp.c

```diff
@@ -1137,6 +1137,7 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, gpsRescueConfig()->throttleHover);
         sbufWriteU8(dst,  gpsRescueConfig()->sanityChecks);
         sbufWriteU8(dst,  gpsRescueConfig()->minSats);
+        sbufWriteU16(dst, gpsRescueConfig()->targetLandingAltitudeM);
```

Comment:
MSP for this should not be added at this point - this only gets added after the changes have been released, to allow for the possibility of changes being made based on user feedback.

Reply:
ok, done.